src/sentry/constants.py

```diff
@@ -29,7 +29,7 @@ def get_all_languages() -> List[str]:
     return results
 
 
-MODULE_ROOT = os.path.dirname(__import__("sentry").__file__)
+MODULE_ROOT = os.path.dirname(cast(str, __import__("sentry").__file__))
```

Comment:
`__file__` is _technically_ nullable -- the `cast` is the shortest code to fix this (personally I'd use `import sentry`; assert sentry.__file__ is not None; # ...` -- but I'm not certain about import cycles here